src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/modele/NamedURL.java

```diff
@@ -0,0 +1,3 @@
+package school.hei.patrimoine.visualisation.swing.ihm.google.modele;
+
+public record NamedURL(String name, String URL) {}
```

Comment:
type URL exists

Reply:
we can change url attribute type to URL because the type of linkIdParser.apply(rawText.trim()) is not a standard type of url but doc file id